### ISSUE TITLE
HABmin doesn't show zwave node IDs

### ISSUE THREAD

**Issue by TheKorn2** [openhab/org.openhab.ui.habmin#266]
This is a continuation of https://github.com/openhab/openhab2-addons/issues/2226 .

Upgraded to latest branch available via apt, 2.0.0 2017-05-08...  still no zwave device IDs being shown in habmin.

![habmin_no_node_ids_again](https://cloud.githubusercontent.com/assets/6579539/25930685/45b1e5c8-35cd-11e7-846a-fc3cc8cec7b1.png)

**Comment by cdjackson** [comment]
As already said in the other issue, this has changed in the development binding and I don't want to add it just to have to remove it again. The development binding now has this as a configuration parameter and it shows up in HABmin as I showed you in the other issue.

**Comment by TheKorn2** [comment]
Chris, I guess I don't understand your answer.  The repository I added was the snapshot repository, as detailed here:  http://docs.openhab.org/installation/linux.html

If that's not the repository that I should have added _then which repository I should be using if not the nightly snapshot repository!?!???!?_

**Comment by cdjackson** [comment]
Sorry - I’m not talking about the snapshot - I’m talking about another branch (i.e. the development branch) which includes new additions to the binding.

There’s a long thread here -:
https://community.openhab.org/t/oh2-z-wave-refactoring-and-testing-and-security/21653 <https://community.openhab.org/t/oh2-z-wave-refactoring-and-testing-and-security/21653>

It will be a month or two before this gets into the snapshot.

**Comment by TheKorn2** [comment]
OOOooooohhhh.  OK, thanks.  Apologies!
